scripts/retry-kubectl.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I actually don't know why this file isn't being shown as being renamed, since it has been moved with `git mv`.

Reply:
I think replacing the original file with a symlink confuses GH.
🤷🏻